docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,430 @@
+)
+USER_DATASET_URL = f"{dataset_repository_url}/user_acquisiton_data.csv"
+NOW_TS = 1708529056
+EXECUTOR_DATA = {CONTEXT_COMMON: {CONTEXT_COMMON_NOW: NOW_TS}}
+```
+
+(If you're waiting something you're executing to finish, you can always find interesting reading in [VectorHub](https://superlinked.com/vectorhub/).)
+
+## Read and explore our dataset
+
+Now that our dataset's imported, let's take a closer look at it:
+
+```python
+NROWS = int(os.getenv("NOTEBOOK_TEST_ROW_LIMIT", str(sys.maxsize)))
+user_df: pd.DataFrame = pd.read_csv(USER_DATASET_URL, nrows=NROWS)
+print(f"User data dimensions: {user_df.shape}")
+user_df.head()
+```
+
+We have 8000 users and (as we can see from the first five rows) 4 columns of data:
+
+![first 5 rows of our dataset](/user_id-sign_up_date-ad_creative-activity.png)
+
+To understand which ad creatives generated how many signups, we create a DataFrame:
+
+```python
+pd.DataFrame(user_df["ad_creative"].value_counts())
+```
+
+which looks like this:
+
+![ad_creatives by count](/ad_creative-count.png)
+
+observation: the influencer (XYZCr$$d) backed ad creatives seem to have worked better - generating many more [signups?] than the August ad creatives.
+
+Now, let's take a look at the distribution of users according to activity level.
+
+```python
+alt.Chart(user_df).mark_bar().encode(
+    alt.X("activity:Q", bin=True, title="Activity count"),
+    alt.Y("count()", title="# of users"),
+).properties(width=600, height=400)
+```
+
+![distribution by activity count](/user_distrib-activity_count.png)
+
+out[6]: 
+The activity distribution is bimodal. Here, the first activity level group may be largely new users, who - because they signed up in the most recent (December) campaign - have less time to accumulate activity than users who signed up earlier.
+Also, this informs NumberSpace parameters.
+
+Now let's examine the distribution of new users per signup date.
+
+```python
+dates: pd.DataFrame = pd.DataFrame(
+    {"date": [str(datetime.fromtimestamp(ts).date()) for ts in user_df["signup_date"]]}
+)
+dates_to_plot = pd.DataFrame(dates.value_counts(), columns=["count"]).reset_index()
+alt.Chart(dates_to_plot).mark_bar().encode(
+    alt.X("date", title="Signup date"), alt.Y("count", title="Number of subscribers")
+).properties(height=400, width=1200)
+```
+
+![new users per signup date](/new_users-signup_date.png)
+
+[observations] First, the distribution confirms that our second campaign (December) works much better than the first (August).
+Second, the jump in signups at 2023-12-21 is due to the second campaign (our data is exclusively campaign-related). To analyze the two campaigns, we need two periods: a first period of 65 days and a second of 185 days seems appropriate.
+Of our 8k users, roughly 2k subscribed in the first campaign, and 6k in the second. "Maybe the second push brought in subscribers intrinsically, but also through spillover to old ads as well - will see that by the ad_creatives..."
+...
```

Comment:
@morkapronczay
I assume the first campaign period is 65 days. 
The 185 days period includes both the first and the second campaign periods? Meaning the second campaign period is the 120 days subsequent to the first 65?
Why do 185 days instead of two subsequent signup periods?

Reply:
1. PeriodTimes are always calculated from the start - it refers to the parameter of the RecencySpace
2. Yes, it is possible they came later (spillover effect meant this exactly)